src/functionalTest/java/uk/gov/hmcts/divorce/legaladvisor/LegalAdvisorMakeDecisionFT.java

```diff
@@ -304,10 +293,10 @@ public void midEventShouldGenerateRefusalOrderWhenAmendApplicationSelected() thr
     private uk.gov.hmcts.ccd.sdk.type.Document uploadDocument() throws IOException {
         var document = caseDocumentAccessManagement.upload(
             idamTokenGenerator.generateIdamTokenForSystem(),
-            serviceAuthenticationGenerator.generate(),
+            serviceAuthenticationGenerator.generate("nfdiv_case_api"),
```

Comment:
is this safe @FaisalMoJ @adamg-hmcts ?

Reply:
I have removed hardcoded service name and getting it through config